from .handle_syscall import handle_syscall
from .errors import get_errno, error_raised